### PR TITLE
Fix unable to import datasets without classification markings, and au…

### PR DESCRIPTION
**Pull request by patstevens4**
…to import food demo dataset

## What does this PR do?
1. Adds new toggle to the create database screen in the Arcade GUI to enable classification markings validation
2. If the toggle is unchecked, the classification field is auto populated to Unclass, and set to read only to prevent users from attempting to put classified data into a marked database without validation or data access enforcement.
3. Auto import the included Food Demo dataset in arcade
4. Fixed the current database classification marking not showing on the query screen

## Motivation
Supporting demos

## Related issues
Closes #2477
Relates to #2481

## Additional Notes
This needs to be tested with the related DF PR: https://github.com/raft-tech/data-fabric/pull/2482

## Setup
1. Switch to DF PR branch locally https://github.com/raft-tech/data-fabric/pull/2482

2. Deploy DF

```
./hacks/df_create_kind.sh
export GHP_USERNAME=your_github_username_goes_here
export GHP_SECRET=your_github_pat_goes_here
./hacks/df_deploy.sh
```

3. Build arcade locally

```
 docker buildx build --platform linux/arm64 --load -t raft-tech/arcadedev ./
```

4. load into kind

```
 kind load docker-image raft-tech/arcadedev  --name data-fabric
```

5. Update image in arcade's replicaset

6. Kill the pod to force it to update if needed

7. Check the logs for arcade, you ought to see a log output like this

```
│ 2023-11-14 13:53:00.013 INFO  [ArcadeDBServer] <ArcadeDB_0> Food demo database not found, creating...                                                                                                      │
│ 2023-11-14 13:53:00.247 INFO  [SourceDiscovery] <ArcadeDB_0> Analyzing url: file://demo-data/FoodData_Central_foundation_food_json_2022-10-28.json...                                                      │
│ Parsed 46,067 (9,213/sec 81%) 0 records (0/sec) 3,038 vertices (607/sec) 14,639 edges (2,927/sec 0 skipped) 0 linked edges (0/sec 0%) updated documents 0 (0%)                                             │
│ Parsed 56,724 (10,497/sec 100%) 0 records (0/sec) 3,628 vertices (581/sec) 17,894 edges (3,207/sec 0 skipped) 0 linked edges (0/sec 0%) updated documents 0 (0%)
```

8. Log into the arcade GUI http://arcadedb.localhost/

9. Confirm the food demo dataset is available. It is probably already selected

10. Confirm the following query succeeds and returns graph data.

```
select *, bothE() as `@edges` from `Composite` limit 30
```

### CODE REVIEW

File: engine/src/main/java/com/arcadedb/database/DocumentValidator.java [hunk 1]
```diff
@@ -58,6 +58,12 @@ public static void verifyDocumentClassificationValidForDeployment(String toCheck
 
   public static void validateClassificationMarkings(final MutableDocument document, 
           SecurityDatabaseUser securityDatabaseUser) {
+
+    // Skip validation checks if classification validation is disabled for the database
+    if (!document.getDatabase().getSchema().getEmbedded().isClassificationValidationEnabled()) {
```
Comment (Brancucci) [inline engine/src/main/java/com/arcadedb/database/DocumentValidator.java:63]: Could this ever be null?

File: server/src/main/java/com/arcadedb/server/ArcadeDBServer.java [hunk 2]
```diff
@@ -180,6 +183,36 @@ public synchronized void start() {
...
+    if (getDatabaseNames() == null || !getDatabaseNames().contains("Food_Demo")) {
+      LogManager.instance().log(this, Level.INFO, "Food demo database not found, creating...");
+      Database database = createDatabase("Food_Demo", READ_WRITE, "U", "admin", true, false);
+
+      try {
+        InputStream inputStream = PostServerCommandHandler.class.getResourceAsStream("/importFoodDemoDatasetCommand.txt");
+        String data = readFromInputStream(inputStream);
+        database.command("sql", data);
+      } catch (Exception e) {
+        e.printStackTrace();
```
Comment (Brancucci) [inline server/src/main/java/com/arcadedb/server/ArcadeDBServer.java:196]: Do we want this captured by the logger?

File: engine/src/main/java/com/arcadedb/database/DocumentValidator.java (hunk 1 above)
Comment (patstevens4) [reply engine/src/main/java/com/arcadedb/database/DocumentValidator.java:63]: Shouldn't be, there's a default value set

File: server/src/main/java/com/arcadedb/server/ArcadeDBServer.java (hunk 2 above)
Comment (patstevens4) [reply server/src/main/java/com/arcadedb/server/ArcadeDBServer.java:196]: nope, fixed

**Review by Brancucci — APPROVED** [review]

**Pull request merged**